client/src/backend/components/project/Content/index.js

```diff
@@ -176,30 +160,6 @@ export class ProjectContent extends PureComponent {
     this.toggleBlockVisibility(block, false);
   };
 
-  mapRawDragResult(result) {
-    if (!result.destination || !result.source) return;
-    const {
-      draggableId: type,
-      source: { index: sourceIndex, droppableId: sourceList },
-      destination: { index: destinationIndex, droppableId: destinationList }
-    } = result;
-    return { type, sourceIndex, sourceList, destinationIndex, destinationList };
-  }
-
-  updateBlockPosition(block, position) {
-    const attributes = Object.assign({}, block.attributes, {
-      position: this.adjustedPosition(position)
-    });
-    return Object.assign({}, block, { attributes });
-  }
-
-  adjustedPosition(position) {
-    const max = this.state.blocks.length;
-
-    if (position >= max) return "bottom";
-    return position;
-  }
-
   handleAddEntity = type => {
     return this.insert(type, 1);
```

Comment:
This has to be updated too since `insert` was removed.  Needs to make a pending block of whatever type and prepend to currentBlocks.  Or maybe just push to currentBlocks?